lib/react_on_rails/configuration.rb

```diff
@@ -266,6 +256,34 @@ def raise_missing_components_subdirectory
 
       raise ReactOnRails::Error, msg
     end
+
+    def using_shakapacker_6?
+      shakapacker_major_version = ReactOnRails::WebpackerUtils.shakapacker_version_as_array[0]
+
+      shakapacker_major_version == 6
+    end
+
+    def shakapacker_precompile?
+      return Webpacker.config.webpacker_precompile? if using_shakapacker_6?
+
+      Webpacker.config.shakapacker_precompile?
+    end
+
+    def shakapacker_precompile_true_message
```

Comment:
We should rename this to something like "compile_command_conflict_message".

Reply:
Done